### PR TITLE
feat: initial copy suggestions

### PR DESCRIPTION
**Pull request by filipviz**
Copy review/suggestions throughout website (not including landing page – will do soon).

Some notes:

- We need to do a better job of communicating Defifa's value proposition: permissionless onchain prediction games with no third parties.
- Tried showing to someone who doesn't use crypto. They asked "why do you need to vote"?
- We should minimize crypto terminology when possible. Instead of using "mint", just say they "buy" NFTs.
- Minimize use of these words: scorecard, outcomes, mint.
- Decentralized, uncensorable, unstoppable prediction games on Ethereum.

### CODE REVIEW

File: src/pages/about/index.tsx [hunk 1]
```diff
@@ -14,73 +14,105 @@ const Explanation = () => {
...
+          Minting NFTs loads a shared reward pot, and the winning teams get more
+          of that pot when the game ends.
         </p>
         <p className="my-4">
-          Which NFTs are "right" is decided by onchain voting. Once a game
-          finishes, anyone can submit scores to determine how much of the pot
-          will go to the NFTs for each prediction. At least 50% of the NFTs for
-          at least 50% of the predictions have to approve a set of scores –
-          otherwise, people can reclaim their ETH from the pot.
+          Which teams "win" is determined by onchain voting. Once NFT minting
```
Comment (i001962) [inline src/pages/about/index.tsx:29]: wins vs win here?

File: src/pages/about/index.tsx [hunk 2]
```diff
@@ -14,73 +14,105 @@ const Explanation = () => {
...
-            reclaim their ETH from the pot.
+            <b>3. Refunds:</b> A refund window opens if the game's creator set
+            one up. During this window, anyone can redeem their NFT to reclaim
+            their ETH from the pot. <b>This stage is optional.</b>
           </li>
           <li className="my-2">
-            <b>4. Earn:</b> If a set of scores is approved, people will be able
-            to burn their NFTs to claim their share of the rewards based on
-            their predictions.
+            <b>4. Score:</b> Once a game ends, anyone can score the game's
```
Comment (i001962) [inline src/pages/about/index.tsx:65]: The timing here for submitting a scorecard is variable and set at game deploy (currently not in ux). We are defaulting to as soon as Game Starts (mint and refund end) anybody may submit a score.

File: src/pages/about/index.tsx [hunk 3]
```diff
@@ -14,73 +14,105 @@ const Explanation = () => {
...
+            results to determine how much of the pot will go to each team. At
+            least 50% of teams need to approve a set of scores by majority vote
+            – otherwise, ETH stays in the pot.
+          </li>
+          <li className="my-2">
+            <b>5. Earn:</b> Once scores are approved, people can burn their NFTs
+            to claim a portion of their team's ETH from the pot. They also get a
+            portion of any Defifa/Juicebox governance tokens earned by the game.
```
Comment (i001962) [inline src/pages/about/index.tsx:73]: is it both dififa and jbx here? or just defifa?

File: src/pages/about/index.tsx [hunk 4]
```diff
@@ -14,73 +14,105 @@ const Explanation = () => {
...
-      <h2 className="text-4xl md:text-4xl text-center font-medium mt-12 mb-6">FAQ</h2>
+      <h2 className="text-4xl md:text-4xl text-center font-medium mt-12 mb-6">
+        FAQ
+      </h2>
       <div className="mx-auto max-w-4xl mb-16">
+        <p className="my-4 font-bold italic text-center text-lg">
+          Why do we need to vote?
+        </p>
+        <p className="my-4" style={{ textIndent: 50 }}>
+          Defifa allows anyone to make prediction games without relying on third
```
Comment (i001962) [inline src/pages/about/index.tsx:86]: This sounds a bit misleading. 'without replying on third party' when in fact players are relying on N parties within their game (or their delegates). Not sure I have a better suggestions atm... 

Consider altering the next sentence too as onchain voting does not mean decentralization. One may deploy a game with 'a' scorekeeper/referee/delegate that has dungeon master level abilities.

File: src/pages/about/index.tsx (hunk 2 above)
Comment (filipviz) [reply src/pages/about/index.tsx:65]: Fixing

File: src/pages/about/index.tsx (hunk 3 above)
Comment (filipviz) [reply src/pages/about/index.tsx:73]: Both (see [this message](https://discord.com/channels/1040017278302965761/1070415506777063444/1121530134189576264)).

File: src/pages/about/index.tsx (hunk 4 above)
Comment (filipviz) [reply src/pages/about/index.tsx:86]: @i001962 good point. Removing "decentralized". I think people generally understand the context of "third-party" here (no centralized service).

If you feel strongly about this I can change it.

**Review by i001962 — APPROVED** [review]

**Pull request merged**